cpu/riscv/extensions/BUILD

```diff
@@ -0,0 +1,972 @@
+# Constraint settings describing RISC-V extensions.
+licenses(["notice"])
```

Comment:
I think you can leave this out.  It is a no-op for Bazel and now internally with Blaze.